docs/operate/reference/components/camera/webcam.md

```diff
@@ -151,6 +151,11 @@ system_profiler SPCameraDataType
 
 The Unique ID displayed for each camera is the `video_path`.
 
+If you are using macOS version 15.x.x Sequoia or later, you need to give `viam-server` permissions to access webcams.
+When you run `viam-server` for the first time, a pop-up message will ask for camera access.
+Click **Allow**.
+Go to **System Settings** > **Privacy & Security** > **Camera** and check the toggle next to `viam-server`.
```

Comment:
```suggestion
To confirm settings, you can go to **System Settings** > **Privacy & Security** > **Camera** and check that the toggle next to `viam-server` is set to enable access.
```
Because "check the toggle" could be interpreted as the act of ticking a checkbox, and because no action is needed theoretically